source/plugin/data/keys.rst

```diff
@@ -87,18 +94,17 @@ Or, if you use Java 8, you're able to shorten the line with lambda expressions:
     }
 
 Note that in both cases we need to make sure our passed function can handle ``null``. You will also notice that no
-check has been performed if the target actually supports the :javadoc:`org.spongepowered.api.data.key.Keys#MAX_HEALTH`
-key. If a target does not support it, the ``transform()`` function will fail and return a ``DataTransactionResult``
-indicating so.
+check has been performed if the target actually supports the :javadoc:`Keys#MAX_HEALTH` key. If a target does not
+support it, the ``transform()`` function will fail and return a ``DataTransactionResult`` indicating so.
 
 Keyed Values
 ============
 
 There are cases where you may care about not only the direct value for a Key, but the keyed value
 encapsulating it. In that case, use the ``getValue(key)`` method instead of ``get(key)``. You will receive an
-object inheriting from :javadoc:`org.spongepowered.api.data.value.BaseValue` which contains a copy of the original
-value. Since we know that current health is a :javadoc:`org.spongepowered.api.data.value.mutable.MutableBoundedValue`,
-we can find out what is the minimum possible value and set our target's health just a tiny bit above that.
+object inheriting from :javadoc:`BaseValue` which contains a copy of the original value. Since we know that current
+health is a :javadoc:`MutableBoundedValue`, we can find out what is the minimum possible value and set our target's
```

Comment:
If you chop out "what is" it becomes more readable. (The "is" is in the wrong place anyway).